### PR TITLE
[issue #71]register tags info to server center

### PR DESCRIPTION
**Pull request by GuoYL123**
send tags to server center

### CODE REVIEW

File: spring-cloud-huawei-router-client/src/main/java/com/huaweicloud/router/client/rest/RouterRestTemplateIntercptor.java [hunk 1]
```diff
@@ -43,6 +42,7 @@
   public ClientHttpResponse intercept(HttpRequest httpRequest, byte[] bytes,
       ClientHttpRequestExecution clientHttpRequestExecution) throws IOException {
     RouterTrackContext.setServiceName(httpRequest.getURI().getHost());
+    httpRequest.getHeaders();
```
Comment (liubao68) [inline spring-cloud-huawei-router-client/src/main/java/com/huaweicloud/router/client/rest/RouterRestTemplateIntercptor.java]: No use code?

File: spring-cloud-huawei-servicecomb-discovery/src/main/java/com/huaweicloud/servicecomb/discovery/discovery/ServiceCombDiscoveryClientConfiguration.java [hunk 2]
```diff
@@ -51,6 +52,12 @@ public ServiceCombDiscoveryProperties serviceCombProperties() {
     return new ServiceCombDiscoveryProperties();
   }
 
+  @Bean
+  @ConditionalOnMissingBean
+  public TagsProperties tagsProperties() {
+    return new TagsProperties();
+  }
```
Comment (liubao68) [inline spring-cloud-huawei-servicecomb-discovery/src/main/java/com/huaweicloud/servicecomb/discovery/discovery/ServiceCombDiscoveryClientConfiguration.java]: How we configure tags in servicecomb? Can do it in spring boot the same in application.yml?

Comment (GuoYL123) [reply spring-cloud-huawei-servicecomb-discovery/src/main/java/com/huaweicloud/servicecomb/discovery/discovery/ServiceCombDiscoveryClientConfiguration.java]: servicecomb load config by com.netflix.config.ConcurrentCompositeConfiguration .
In spring-cloud-huawei it use @ConfigurationProperties to load config. 
It's hard to say which one is better.
But is this the better to keep the same as previous code?

File: spring-cloud-huawei-router-client/src/main/java/com/huaweicloud/router/client/rest/RouterRestTemplateIntercptor.java (hunk 1 above)
Comment (GuoYL123) [reply spring-cloud-huawei-router-client/src/main/java/com/huaweicloud/router/client/rest/RouterRestTemplateIntercptor.java]: delete it

**Pull request merged**